FEM.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import random
+import string
+import os
+import time
+
+class FileMonitor:
+    def __init__(self, file1, file2, logfile):
+        self.file1 = file1
+        self.file2 = file2
+        self.logfile = logfile
+        self.count1 = 0
+        self.count2 = 0
+
+    async def random_writer(self):
+        while True:
+            with open(self.file1, 'a') as f1, open(self.file2, 'a') as f2:
+                if random.random() > 0.5:
```

Comment:
@Mayur1Mtech Can we improve this logic? Since it is not right and will not be able to ensure a 50% probability

Reply:
I tried that logic:
random.choices(population, weights=50, k=1)
I will try it again